definitions/firefox_ios.toml

```diff
@@ -197,6 +235,16 @@ owner = "vsabino@mozilla.com"
 
 [data_sources]
 
+[data_sources.active_users_aggregates]
+from_expression = """(
+    SELECT *, NULL as client_id
```

Comment:
Adding the `NULL as client_id` here is a hack; I'm not sure if there's a better way to do this. We want to use `active_users_aggregates` for these metrics because we have bigquery-etl infrastructure that ensures the values in this table are correct. However, this table doesn't include `client_id`, and we don't need it for calculating DAU.